src/types/onyx/ReportAction.ts

```diff
@@ -89,7 +91,16 @@ type ReportActionBase = {
     childCommenterCount?: number;
     childLastVisibleActionCreated?: string;
     childVisibleActionCount?: number;
+    parentReportID?: string;
```

Comment:
Please add the comments for onyx properties that were added in this PR, at least some of them 😅